packages/container/lib/container.js

```diff
@@ -113,7 +113,7 @@ Container.prototype = {
    @param {String} fullName optional key to reset; if missing, resets everything
    */
   reset(fullName) {
-    if (arguments.length > 0) {
+    if (fullName !== undefined) {
```

Comment:
we prefer arity for such things, as accidentally passing undefined around is quite common.